demisto_sdk/__main__.py

```diff
@@ -880,6 +883,83 @@ def find_dependencies_command(**kwargs):
         print_error(str(exp))
 
 
+# ====================== openapi-codegen ====================== #
+@main.command(name="openapi-codegen",
+              short_help='''Generate an XSOAR integration with the OpenAPI(Swagger) generator tool.''')
+@click.help_option(
+    '-h', '--help'
+)
+@click.option(
+    '-i', '--input_file', help='The swagger file to load', required=True)
```

Comment:
```suggestion
    '-i', '--input_file', help='The JSON Swagger file to load', required=True)
```

Reply:
only json at the moment. I can accept a yaml and just convert it to json

Reply:
im asking because i see in https://github.com/demisto/demisto-sdk/pull/629/files#diff-bb0b8011133ced87e4afb0f4187f070aR272 you are trying to load yml ?

Reply:
removed that

Reply:
ok lets change here to `The Swagger JSON file to load`